### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'xpath'

### ISSUE THREAD

**Issue by deiferni** [4teamwork/docxcompose#5]
Reproduce the issue:
- go to https://spv.onegovgever.ch/sitzungen/committee-8/meeting-169/view
- click `Protokollauszug generieren` for any agenda item

https://sentry.4teamwork.ch/sentry/onegov-gever/issues/4813/

```
AttributeError: 'NoneType' object has no attribute 'xpath'
(9 additional frame(s) were not displayed)
...
  File "home/zope/theta.4teamwork.ch/55-spv.onegovgever.ch/src/opengever.core/opengever/meeting/mergetool.py", line 45, in add
    remove_property_fields=self._remove_property_fields)
  File "docxcompose/composer.py", line 37, in append
    self.insert(index, doc, remove_property_fields=remove_property_fields)
  File "docxcompose/composer.py", line 56, in insert
    self.add_styles(doc, element)
  File "docxcompose/composer.py", line 173, in add_styles
    num_ids = xpath(style_element, './/w:numId/@w:val')
  File "docxcompose/utils.py", line 23, in xpath
    return element.xpath(xpath_str, namespaces=NS)

AttributeError: 'NoneType' object has no attribute 'xpath'
```

**Comment by deiferni** [comment]
[footer.docx](https://github.com/4teamwork/docxcompose/files/1574756/footer.docx)
[header.docx](https://github.com/4teamwork/docxcompose/files/1574757/header.docx)
[neuer-spielplatz.docx](https://github.com/4teamwork/docxcompose/files/1574758/neuer-spielplatz.docx)

**Comment by buchi** [comment]
Fixed in 5fc7433328fa0321f98d78761a445543accc198a
